tests/test_auth.py

```diff
@@ -2,7 +2,7 @@
 # In case of editing this file, make sure the change is propagated to all Databricks SDK codebases
 from databricks.sdk.core import Config
 
-from .conftest import __tests__, raises
+from .conftest import __tests__, raises, set_az_path, set_home
```

Comment:
Is `__tests__` still needed?

Reply:
in line 219 of `test_auth.py`, We are pointing the `PATH` to some garbage place such that It will not find the `az` command. I am currently not passing any argument related to path in `set_az_path`. Because in rest of the cases the path is fixed. thats why I needed the `__tests__`. what are your suggestions regarding this?

Reply:
It's alright, more of a nit not to import things that aren't used, but if it is still used I just missed it. We can leave as-is.